Compressonator/Applications/_Plugins/CImage/EXR/CMakeLists.txt

```diff
@@ -18,22 +18,36 @@ target_sources(EXR
                ./EXR.cpp
                )
 
+find_path(ILMBASE_INCLUDE_DIR OpenEXR/half.h)
+find_path(OPENEXR_INCLUDE_DIR OpenEXR/ImfNamespace.h)
+if(ILMBASE_INCLUDE_DIR)
+    target_include_directories(CompressonatorCLI-bin
```

Comment:
You should not define include directories for the `CompressonatorCLI-bin` target in a unrelated CMakeLists.txt, but only in `Compressonator/Applications/CompressonatorCLI/CMakeLists.txt `.